prisma/schema.prisma

```diff
@@ -46,9 +46,43 @@ model User {
   name          String?
   email         String?   @unique
   emailVerified DateTime? @map("email_verified")
+  username      String?   @unique
+  password      String?
   image         String?
   accounts      Account[]
   sessions      Session[]
+  roles         UserRole[]
+  archived      Boolean   @default(false)
+  createdAt     DateTime  @default(now()) @map("created_at")
+  updatedAt     DateTime? @map("updated_at")
+  accessStatus  AccessStatus  @default(PENDING) @map("access_status")
 
   @@map("users")
 }
+
+model Role {
```

Comment:
nevermind about the role here, i just implemented to test role-based authorize but forgot to revert. I will refactor it in task AL-32